stories/notifications.stories.js

```diff
@@ -205,4 +206,7 @@ storiesOf( 'Global/Notifications', module )
 			}
 			type="win-stats"
 		/>
+	) )
+	.add( 'User Input Settings', () => (
+		<UserInputSettings />
```

Comment:
This needs to at least be wrapped with `WithTestRegistry`, it's currently broken otherwise.
![image](https://user-images.githubusercontent.com/1621608/99292432-e1be4980-2849-11eb-89a3-4d9437478fa9.png)